### PR TITLE
Deploy 0.112.0 to mainnet-na and mainnet-na-citus.

### PR DESCRIPTION
**Pull request by mgoelswirlds**
**Description**:

Deploy 0.112.0 to mainnet-na and mainnet-na-citus. Adding the rest-java db password to the sealed secret.

**Related issue(s)**:

Fixes #

**Notes for reviewer**:

**Checklist**

- [ ] Documented (Code comments, README, etc.)
- [ ] Tested (unit, integration, etc.)

### CODE REVIEW

File: clusters/mainnet-na/common/helmrelease.yaml [hunk 1]
```diff
@@ -10,7 +10,7 @@ spec:
       sourceRef:
         kind: HelmRepository
         name: hedera-mirror-node
-      version: '0.111.1'
+      version: '0.112.0'
```
Comment (steven-sheehy) [inline clusters/mainnet-na/common/helmrelease.yaml:13]: We need to include the custom operator image.

File: clusters/mainnet-na/mainnet-citus/helmrelease.yaml [hunk 2]
```diff
@@ -14,7 +14,7 @@ spec:
         - values.yaml
         - values-prod.yaml
         - values-v2.yml
-      version: '0.111.1'
+      version: '0.112.0'
   dependsOn:
     - name: mirror
       namespace: common
```
Comment (steven-sheehy) [inline clusters/mainnet-na/mainnet-citus/helmrelease.yaml:20]: Jesse, can we uncomment the drift detection now?

File: clusters/mainnet-na/common/helmrelease.yaml (hunk 1 above)
Comment (mgoelswirlds) [reply clusters/mainnet-na/common/helmrelease.yaml:13]: I believe @xin-hedera is going to add that.

File: clusters/mainnet-na/mainnet-citus/helmrelease.yaml
[the remark this answers is not in the document]
Comment (jnels124) [reply clusters/mainnet-na/mainnet-citus/helmrelease.yaml:1]: wasn't token reject moved to 0.53.0?

File: clusters/mainnet-na/mainnet-citus/helmrelease.yaml (hunk 2 above)
Comment (jnels124) [reply clusters/mainnet-na/mainnet-citus/helmrelease.yaml:20]: yes we should re-enable. No more manual changes should be done here

Comment (jnels124) [reply clusters/mainnet-na/mainnet-citus/helmrelease.yaml:20]: Also remove the coordinator instances override please

File: clusters/mainnet-na/mainnet-citus/helmrelease.yaml
[the remark this answers is not in the document]
Comment (steven-sheehy) [reply clusters/mainnet-na/mainnet-citus/helmrelease.yaml:1]: Looks like it, though the release deliverables was not updated in monday.

**Review by jnels124 — APPROVED** [review]
LGTM

**Pull request merged**